easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20170420-R1_rungms_fix_PPN_not_initialized.patch

```diff
@@ -0,0 +1,10 @@
+--- rungms.orig	2017-07-19 16:53:29.325530604 +0000
```

Comment:
Please include a short description above this line + name of author or reference to upstream patch.